handler/src/main/java/io/netty/handler/ipfilter/IpSubnetFilter.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright 2020 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.ipfilter;
+
+import io.netty.channel.Channel;
+import io.netty.channel.ChannelHandler.Sharable;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.util.internal.ObjectUtil;
+
+import java.net.InetSocketAddress;
+import java.net.SocketAddress;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * <p>
+ * This class allows one to filter new {@link Channel}s based on the
+ * {@link IpSubnetFilter}s passed to its constructor. If no rules are provided, all connections
+ * will be accepted.
+ * </p>
+ * <p>
+ * If you would like to explicitly take action on rejected {@link Channel}s, you should override
+ * {@link #channelRejected(ChannelHandlerContext, SocketAddress)}. </p>
```

Comment:
This should say `AbstractRemoteAddressFilter#channelRejected(ChannelHandlerContext, SocketAddress)` for the javadoc reference to work.

Reply:
Actually this is coming from https://github.com/netty/netty/blob/4be9387151d9702a17da56cffaa6e5e4c92245fd/handler/src/main/java/io/netty/handler/ipfilter/RuleBasedIpFilter.java#L32